crypto/mem_sec.c

```diff
@@ -57,10 +56,6 @@ int CRYPTO_secure_malloc_init(size_t size, int minsize)
 #ifdef IMPLEMENTED
     int ret = 0;
 
-    if (too_late)
-        return ret;
```

Comment:
Not sure about the removal of `too_late`.  @richsalz, did you notice this?  Opinion?